### PR TITLE
Test for gene search results

### PR DESCRIPTION
**Pull request by creilly8**
## Description
Requested test in issue #2036 runs locally but different iterations failed on CI three times. Test moved to manual integration file to test locally. 

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

**Comment by xzhou82** [comment]
as i have said, string2variant() won't work for hg38-test, due to the fact that fasta is missing.
just disable those tests and change it to CI

**Comment by creilly8** [comment]
> as i have said, string2variant() won't work for hg38-test, due to the fact that fasta is missing. just disable those tests and change it to CI

@xzhou82, That test failed in a separate file, in different iterations, three times: 
1. https://github.com/stjude/proteinpaint/actions/runs/10475622164
2. https://github.com/stjude/proteinpaint/actions/runs/10474918783
3. https://github.com/stjude/proteinpaint/actions/runs/10474798769

The last commit is moving the test from an `integration.spec` file to the`spec-manual` to avoid a second manual test file.

File: client/dom/test/genesearch.integration.spec-manual.ts [hunk 1]
```diff
@@ -1,14 +1,42 @@
 import tape from 'tape'
 import { string2variant } from '../genesearch.ts'
 import { hg38 } from '../../test/testdata/genomes'
```
Comment (xzhou82) [inline client/dom/test/genesearch.integration.spec-manual.ts:3]: this hg38 genome obj is not the hg38-test, that's why it never works on CI

**Review by xzhou82 — APPROVED** [review]
ok to merge now if not easy to fix it

File: client/dom/test/genesearch.integration.spec.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import tape from 'tape'
 import { string2variant } from '../genesearch.ts'
-import { hg38 } from '../../test/testdata/genomes'
+import { hg38, hg38_test } from '../../test/testdata/genomes'
```
Comment (xzhou82) [inline client/dom/test/genesearch.integration.spec.ts:3]: do not import this testdata. query `/genomes` instead

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**